conjur/data_object/create_token_data.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+"""
+VariableData module
+
+This module represents the DTO that holds the params the user passes in.
+We use this DTO to build the hostfactory create token request
+"""
+
+
+# pylint: disable=too-few-public-methods
+from datetime import datetime, timedelta
+
+
+class CreateTokenData:
+    """
+    Used for organizing the params the user passed in to execute the CreateToken command
+    """
+
+    def __init__(self, **arg_params):
```

Comment:
why kwargs and not just simple parameters? is the input dynamically changes?

Reply:
I think we should always be as explicit as possible and avoid kwargs